features/step_definitions/support_notice_steps.rb

```diff
@@ -0,0 +1,84 @@
+### GIVEN
+
+Given "there are no support notices" do
+  SupportNotice.delete_all
+end
+
+### WHEN
+
+When /^an admin creates an?( active)?(?: "([^"]*)")? support notice$/ do |active, notice_type| # rubocop:disable Cucumber/RegexStepName
+  step %{I am logged in as a "support" admin}
+  visit(new_admin_support_notice_path)
+  fill_in("support_notice_notice", with: "This is some support notice text")
+  case notice_type
+  when "caution"
+    choose("support_notice_support_notice_type_caution")
+  when "error"
+    choose("support_notice_support_notice_type_error")
+  else
+    choose("support_notice_support_notice_type_notice")
+  end
+  check("support_notice_active") if active.present?
+  click_button("Create Support Notice")
+  step %{I should see "Support Notice successfully created."} if active.present?
+end
+
+When "an admin deactivates the support notice" do
+  step %{I am logged in as a "support" admin}
```

Comment:
logging in steps like this slow down the tests. could you remove them where they're not needed?